containers/ws/release.sh

```diff
@@ -8,24 +8,52 @@ if [ -z "${RUNC:-}" ]; then
     }
 fi
 
-$RUNC build -t quay.io/cockpit/ws:release containers/ws
+# only build on host architecture per default
+if [ -z "${ARCHS:-}" ]; then
+    case $(uname -m) in
+        x86_64) ARCHS="amd64" ;;
+        arm64) ARCHS="arm64" ;;
+    esac
+fi
+for arch in $ARCHS; do
+    $RUNC buildx build --platform="linux/${arch}" -t "quay.io/cockpit/ws:release-${arch}" containers/ws
+done
 
 # smoke test
-name=ws-release
-$RUNC run --name $name -p 19999:9090 -d quay.io/cockpit/ws:release
-until curl --fail --show-error -k --head https://localhost:19999; do
-    sleep 1
+first_arch=$(echo "${ARCHS}" | cut -d " " -f 1)
+test_container=ws-release
+$RUNC run --name "${test_container}" -p 19999:9090 -d "quay.io/cockpit/ws:release-${first_arch}"
```

Comment:
This `first_arch` feels too arbitrary to me. At least on my laptop I cannot actually run arm64 binaries -- perhaps podman has some integration with qemu-system-arm or so, but I don't have that installed. Either the GitHub runners can do that, then the smoke test should run for all images. Or this should hardcode `release-amd64` as that's then the only one which we can test anyway.